gems/aws-sdk-s3/spec/object/download_file_spec.rb

```diff
@@ -179,53 +182,6 @@ module S3
               ":chunk_size shouldn't exceed total file size."
             )
         end
-
-        context 'file size is chunk size boundary' do
```

Comment:
It seems like we should still keep a similar test.

I think we should likely also have a test that size=min_chunk_size uses single_request.

Reply:
I think the mock that I added will work in all cases. It verifies that there's no range where the bytes are equal. There's already a case here where it's on the boundary, too.

Reply:
1 byte ranges apparently work..